.travis-pub.yml

```diff
@@ -8,21 +8,51 @@ go:
 
 env:
```

Comment:
question: what is this .travis-pub.yml for?  how is it different than the plain .travis.yml?

Reply:
@mew2057   I also had this question, forgot to ask....   they  seem very similar.

Reply:
We can remove the `.pub`, it  was something I made when debugging the CI/CD pipeline  for  Cert  integration.